examples/basics/data_row_metadata.ipynb

```diff
@@ -3,7 +3,9 @@
   {
```

Comment:
<p>Can we change this structure slightly to clearly show we have 2 options as listed below?</p><p> </p><ul><li>Option 1: .....</li><li>Option 2: ....</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/587/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>